src/Ship.cpp

```diff
@@ -1259,6 +1260,22 @@ void Ship::StaticUpdate(const float timeStep)
 				DoDamage(damage);
 			}
 		}
+		// play some metal stressing sound when the player acceleration changes too rapidly
+		if (IsType(ObjectType::PLAYER)) {
```

Comment:
If this will only work for the player, it seems better to move this to the player's Static Update
https://github.com/pioneerspacesim/pioneer/blob/44781b7572bbd0c25019132cc165795de2dae696/src/Player.cpp#L295

Reply:
Done.

Reply:
Done.